coffee/src/templates/filter.hbs

```diff
@@ -14,21 +14,14 @@
 
 
 {{#if showScenarioGroup}}
-  <div class="{{#if showAgrCommDevSelector}}scenario-container highlight-section{{/if}}">
   {{#if showScenarioSelector}}
     {{addSubViewTo thisView 'ScenarioSelectorView' filter=filter}}
   {{/if}}
-  {{#if showAgrCommDevSelector}}
-    {{addSubViewTo thisView 'LevelSelectorAgrCommDevView' filter=filter}}
-  {{/if}}
   </div>
```

Comment:
I think we need to remove line 20 if we're removing the opening div at line 17, if this is it's closing counterpart